src/main/resources/openapi/specs/api.kuflow.com/v1/openapi.yaml

```diff
@@ -372,7 +373,7 @@ paths:
       description: |
         Allow to save an element i.e., a field, a decision, a form, a principal or document.
 
-        If is a document you need to pass a document reference using the 'id' attribute.
+        If is a document you need to pass a document reference using the 'uri' attribute.
```

Comment:
```suggestion
       In the case of document type elements, this method only allows references to be made to other existing document type elements for the purpose of copying that file into the element. To do this you need to pass a reference to the document using the 'uri' attribute. In case you want to add a new document, please use the corresponding API method.
```

Reply:
😍